environment.py

```diff
@@ -137,13 +142,23 @@ def remote_cmd(cmd, host=None, ignore_rc=False, **kwargs):
         # the 'context' object can basically hold whatever we want.
         # if we stash the result from Ansible, we can inspect it or log it
         # later
-        context.result = ansible.runner.Runner(
-                 module_name=cmd,
-                 inventory=inventory,
-                 pattern=host,
-                 #remote_user=remote_user,
-                 **kwargs
-        ).run()
+        if async:
+            context.result, poller = ansible.runner.Runner(
+                     module_name=cmd,
+                     inventory=inventory,
+                     pattern=host,
+                     #remote_user=remote_user,
+                     **kwargs
+            ).run_async(context.test_timeout)
+            return poller
+        else:
+            context.result = ansible.runner.Runner(
+                     module_name=cmd,
+                     inventory=inventory,
+                     pattern=host,
+                     #remote_user=remote_user,
+                     **kwargs
+            ).run()
```

Comment:
pep8 will not raise warning for these lines. But it is OK for me to update and during my check there are several style problem exist in the code too. Will format a patch for it. Thanks for reminding.